Gruntfile.js

```diff
@@ -78,6 +78,9 @@ module.exports = function(grunt) {
                   [
                      'babelify',
                      {
+                        plugins: [
+                           '@babel/plugin-proposal-class-properties',
```

Comment:
Is this needed? I'm not seeing that this has any effect:

```bash
git checkout master
git fetch --all
git rebase origin/master
nvm use
npm ci 
grunt clean build
mv dist ../dist-without-plugin
git checkout ismendoza/plugin-proposal-class-properties
nvm use
npm ci
grunt clean build
diff ../dist-without-plugin/silvermine-videojs-chromecast.js ./dist/silvermine-videojs-chromecast.js
```
outputs no difference.

It seems like this plugin is already running (through `@babel/present-env`, I think).